plugins/node/instrumentation-runtime-node/src/types/ConventionalNamePrefix.ts

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+export enum ConventionalNamePrefix {
+  NodeJsRuntime = 'nodejsruntime',
+  V8EnjineRuntime = 'v8jsengineruntime',
```

Comment:
typo: enjine -> engine

regarding the node js metrics, we're going with the shorter name, so just `nodejs` without the `runtime`.
For the v8 there is still deliberation on what name to use, with the possibility of using a generic `jsengine` and have one of the attributes describing it is v8 in this case. Hopefully we can get to an agreement soon 🤞